routemaster/cron.py

```diff
@@ -98,59 +140,22 @@ def __init__(self, app: App) -> None:
         self._terminating = False
         self.app = app
         self.scheduler = schedule.Scheduler()
-        super().__init__(name="cron", daemon=True)
+        super().__init__(name="cron")
 
     def run(self) -> None:
         """Run main scheduling loop."""
         configure_schedule(self.app, self.scheduler, self.is_terminating)
+        logger.info("Starting cron thread")
```

Comment:
Would be nice to have a matching pair for this below -- after the while loop add `logger.info("Exiting cron thread")`

This will be useful in tracing the system and can also be used as a canary as to whether exceptions in the cron thread are leaking out of the above guards.